plugins/kubernetes/app/models/kubernetes/resource_template.rb

```diff
@@ -210,6 +211,12 @@ def set_env
       end
     end
 
+    def set_image_pull_secrets
+      return unless secrets = ENV['KUBERNETES_IMAGE_PULL_SECRETS']
```

Comment:
Is it a problem if we set this for a 1.3.x release of Kubernetes?  If so, we could ask the Kubernetes API for its version.

```
$ curl -sS http://127.0.0.1:9099/version | jq .
{
  "major": "1",
  "minor": "2",
  "gitVersion": "v1.2.4.3+f4ece03d458602",
  "gitCommit": "f4ece03d458602c440b58ce90d0c396e9547e6c8",
  "gitTreeState": "clean"
}
```